game/domain/card.lua

```diff
@@ -13,14 +13,12 @@ function Card:init(specname)
 end
 
 function Card:loadState(state)
-  self:setId(state.id)
   self.specname = state.specname
   self.usages = state.usages
 end
 
 function Card:saveState()
   local state = {}
-  state.id = self:getId()
```

Comment:
Why aren't we saving the card ID anymore?

Reply:
because we never have even given cards ids, plus they have zero use whatsoever. i can put it back in but the cards are going to save nothing still

Reply:
I see. Seems fine to leave them without IDs then.